LLMonFHIR/FHIRSummary/FHIRResourceSummary.swift

```diff
@@ -67,7 +80,22 @@ final class FHIRResourceSummary: Sendable {
     func summarize(resource: FHIRResource, forceReload: Bool = false) async throws -> Summary {
         let resource = try resource.copy()
         try? resource.stringifyAttachements()
-        return try await resourceProcessor.process(resource: resource, forceReload: forceReload)
+
+        var retryCount = 0
+        var summary: Summary
+
+        repeat {
+            summary = try await resourceProcessor.process(resource: resource, forceReload: forceReload || retryCount > 0)
+            retryCount += 1
+
+            if !summary.needsRetry {
+                break
+            }
+
+            try? await Task.sleep(nanoseconds: 500_000_000)
```

Comment:
[There are nice duration-related overloads for `Task.sleep` that we can use.](https://developer.apple.com/documentation/swift/task/sleep(for:tolerance:clock:)#discussion)